notebooks/demo/nx_cugraph_jaccard_movie_recommender.ipynb

```diff
@@ -0,0 +1,533 @@
+    "The actual NetworkX graph object can now be created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9fb98b4e-026c-4e39-8fec-88e53de4eac0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "good_user_movie_G = nx.from_pandas_edgelist(\n",
+    "    good_ratings_df, source=\"userId\", target=\"movieId\", edge_attr=\"rating\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e936c2f0-e718-4041-9bdb-139eb5460187",
+   "metadata": {},
+   "source": [
+    "A user can be chosen at random, and one of their highest rated movies can be picked. The idea is to find movies similar to a movie the user rated highly, then filter out movies the user has already seen and recommend the movie most similar to the highly rated one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "764192e3-d865-4660-aa44-57eaa98eb5f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pick a user and one of their highly-rated movies\n",
+    "user = good_user_ids[321]\n",
+    "user_reviews = good_user_movie_G[user]\n",
+    "highest_rated_movie = max(\n",
+    "    user_reviews,\n",
+    "    key=lambda n: user_reviews[n].get(\"rating\", 0)\n",
+    ")\n",
+    "\n",
+    "print(f\"highest rated movie for {user=} is \"\n",
+    "      f\"{movie_id_name_map[highest_rated_movie]}, \"\n",
+    "      f\"id: {highest_rated_movie}, \"\n",
+    "      f\"rated: {user_reviews[highest_rated_movie]}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1a73938-3ab9-4086-b820-7ae0aa4e8b94",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a list of nodes to compare the user's highest\n",
+    "# rated movie to all other movies in the graph.\n",
+    "ebunch = [(highest_rated_movie, n) for n in good_movie_ids[1:]\n",
+    "          if n != highest_rated_movie]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98aa9dfa-9fee-460b-a3e9-e851570ed77d",
+   "metadata": {},
+   "source": [
+    "The Jaccard Similarity function provides the value used for measuring similarity. Jaccard similarity is described in more detail [here](https://en.wikipedia.org/wiki/Jaccard_index).\n",
+    "\n",
+    "Because the `NX_CUGRAPH_AUTOCONFIG` environment variable was set to `True`, NetworkX will use the Jaccard implementation provided by cuGraph."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aacce305-c05f-4685-8cfe-08e0b48a7b80",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "# Run Jaccard Similarity \n",
+    "jacc_coeffs = list(nx.jaccard_coefficient(good_user_movie_G, ebunch))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e510fe53-d04a-4ef3-b255-ccc2559af9c2",
+   "metadata": {},
+   "source": [
+    "The default NetworkX implementation can be used if specified using the `backend=` kwarg. This will override the backend priority set by the environemnt variable.\n",
```

Comment:
```suggestion
    "The default NetworkX implementation can be used if specified using the `backend=` kwarg. This will override the backend priority set by the environment variable.\n",
```